metabase/metabase
# Hide personal collections and items when configuring click behavior

This is the final part of the epic https://github.com/metabase/metabase/issues/34733

This PR implements
- Hide personal collections and dashboards when configuring click behavior linking to dashboards
- Hide personal collections and questions when configuring click behavior linking to questions

![image](https://github.com/metabase/metabase/assets/1937582/fd364bb5-4eb4-488c-81de-4fa3fe840119)


#### How to verify

Dashboard click behavior

1. Link to questions/dashboards for a question in a public collection
    **expectation**:
    - show only questions/dashboards in public collections
    - search should return only questions/dashboards in public collections
1. Link to questions/dashboards for a question in a personal collection
    **expectation**:
    - show all questions/dashboards
    - search should return all questions/dashboards
